src/blocks/CaseStudyParallax/index.tsx

```diff
@@ -0,0 +1,307 @@
+'use client'
+import * as React from 'react'
+
+import { BackgroundGrid } from '@components/BackgroundGrid'
+import { BackgroundScanline } from '@components/BackgroundScanline'
+import { BlockSpacing } from '@components/BlockSpacing'
+import { Button } from '@components/Button'
+import { Gutter } from '@components/Gutter'
+import { Media } from '@components/Media'
+import { QuoteIconAlt } from '@root/icons/QuoteIconAlt'
+import { Page } from '@root/payload-types'
+import { useResize } from '@root/utilities/use-resize'
+
+import classes from './index.module.scss'
+
+type Props = Extract<Page['layout'][0], { blockType: 'caseStudyParallax' }>
+
+type StickyBlockProps = Props & {
+  currentIndex: number
+}
+
+type QuoteProps = {
+  card: any
+  isVisible?: boolean
+  className?: string
+}
+
+export const QuoteBlock: React.FC<QuoteProps> = props => {
+  const { isVisible, card, className } = props
+  return (
+    <div
+      className={[isVisible && classes.isVisible, 'cols-8 grid', className]
+        .filter(Boolean)
+        .join(' ')}
+      aria-hidden={!isVisible}
+    >
+      <QuoteIconAlt className={classes.quoteIcon} />
+      <div
+        aria-hidden={!isVisible}
+        className={[classes.quote, 'cols-16'].filter(Boolean).join(' ')}
+      >
+        {card.quote}”
+      </div>
+
+      <div
+        aria-hidden={!isVisible}
+        className={[classes.authorWrapper, 'cols-16'].filter(Boolean).join(' ')}
+      >
+        <div className={classes.media}>
+          {typeof card.logo !== 'string' && <Media resource={card.logo} />}
+        </div>
+        <div className={classes.author}>{card.author}</div>
+      </div>
+
+      {typeof card.caseStudy !== 'string' && (
+        <div className={['cols-8 cols-m-4 cols-s-8'].filter(Boolean).join(' ')}>
+          <Button
+            label={'Read the case study'}
+            hideHorizontalBorders
+            appearance={'default'}
+            icon="arrow"
+            className={classes.caseStudyButton}
+            href={`/case-studies/${card.caseStudy.slug}`}
+            el="a"
+            aria-hidden={!isVisible}
+            disabled={!isVisible}
+          />
+        </div>
+      )}
+    </div>
+  )
+}
+
+export const QuoteStickyBlock: React.FC<StickyBlockProps> = props => {
+  const { caseStudyParallaxFields, currentIndex } = props
+
+  if (caseStudyParallaxFields?.cards && caseStudyParallaxFields?.cards?.length > 0) {
+    return (
+      <div className={[classes.stickyBlock, 'grid'].filter(Boolean).join(' ')}>
+        {caseStudyParallaxFields?.cards.map((card, index) => {
+          const isVisible = index === currentIndex
+
+          return (
+            <QuoteBlock isVisible={isVisible} card={card} className={classes.stickyBlockItem} />
```

Comment:
The returned element here needs a key prop I think - might need to wrap it in a `div`

Getting this error in the console:

![Screen Shot 2024-01-25 at 12 16 16 PM](https://github.com/payloadcms/website/assets/35232443/131bc9b0-cc04-4b30-b8a6-c29a97db17b5)